proxy/src/auth/credentials.rs

```diff
@@ -21,7 +21,10 @@ pub enum ComputeUserInfoParseError {
          SNI ('{}') and project option ('{}').",
         .domain, .option,
     )]
-    InconsistentProjectNames { domain: SmolStr, option: SmolStr },
+    InconsistentProjectNames {
+        domain: EndpointId,
+        option: EndpointId,
```

Comment:
Is this also ep id?

Reply:
yeah, it's either the endpoint from the domain (SNI) or from the `options=endpoint%3D...` string